### PR TITLE
Fixes #7755: Backport changes from #7123 to 3.0

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/7755

### CODE REVIEW

**Comment by amousset** [comment]
Takes changes from #914 except renaming parts of the interface.

File: rudder-web/src/main/webapp/templates-hidden/components/ComponentAgentSchedule.html [hunk 1]
```diff
@@ -14,14 +14,14 @@
...
+        <p>By default, the agent runs on all nodes every 5 minutes.</p>
         <p>
-          The high frequency enables fast response time to apply changes and state
+          This high frequency enables fast response times to apply changes and state
           assessment for high-precision drift and compliance reports.
         </p>
         <p>
           You can modify this run interval below, as well as the "splay time" across nodes (a random
-          delay that alters scheduled run time, intended to spread load across nodes).
+          elay that alters scheduled run time, in order to spread load across nodes).
```
Comment (jooooooon) [inline rudder-web/src/main/webapp/templates-hidden/components/ComponentAgentSchedule.html]: Oops, this is missing a "d"

**Comment by jooooooon** [comment]
Thanks for this PR, @amousset. I can vouch for the correctness of the English expressions, but seeing as I wrote them myself, I'm not very impartial.

@VinceMacBuche could you review the actual changes from a technical point of view? (we need @amousset to add that missing "d" before merging, though)

**Comment by VinceMacBuche** [comment]
Seems good to me :) TO merge after feedback treated !

**Comment by amousset** [comment]
PR updated

**Pull request merged**